pkg/cli/start.go

```diff
@@ -452,6 +452,11 @@ func runStartInternal(
 	signalCh := make(chan os.Signal, 1)
 	signal.Notify(signalCh, DrainSignals...)
 
+	// We try to delete old leftover temporary folders
```

Comment:
Could we add to the comment here stating why this cleanup needs to be occur _before_ the call to `exitIfDiskFull`? That lowers the risk of someone coming along and moving it in the future.